Doc/library/argparse.rst

```diff
@@ -1808,7 +1808,9 @@ FileType objects
 Argument groups
 ^^^^^^^^^^^^^^^
 
-.. method:: ArgumentParser.add_argument_group(title=None, description=None)
+.. method:: ArgumentParser.add_argument_group(title, description, \
+                                              [*argument_default], \
+                                              [*conflict_handler])
```

Comment:
Sorry, I was not clear. We need a single `*` deparating keyword-or-positional arguments and keyword-only arguments. This is a Python syntax (even if using brackets for optional parameters is not).

`None` is the right default value for title and description.
```suggestion
.. method:: ArgumentParser.add_argument_group(title=None, description=None, *, \
                                              [argument_default], [conflict_handler])
```